forui/lib/src/widgets/button/button_styles.dart

```diff
@@ -1,7 +1,12 @@
-part of 'button.dart';
+import 'package:flutter/foundation.dart';
+import 'package:flutter/widgets.dart';
+
+import 'package:meta/meta.dart';
+
+import 'package:forui/forui.dart';
 
 /// [FButtonCustomStyle]'s style.
-class FButtonStyles with Diagnosticable {
```

Comment:
What classes do we want to set `final`?

Reply:
I think we should mark all style classes as final.